src/palm-react/hooks/firestore/useFsChannels.ts

```diff
@@ -10,12 +13,45 @@ export type UseFsChannelsReturn = {
 
 const useFsChannels = (): UseFsChannelsReturn => {
   const limit = 5
-
   const [isFetching, setIsFetching] = useState<boolean>(true)
   const [channels, setChannels] = useState<FbChannel[]>([])
 
+  const [recommendedChannelUrls, setRecommendedChannelUrls] = useState<
+    string[] | undefined
+  >(undefined)
+
+  const fetchRecommendedChatUrls = async (): Promise<void> => {
+    try {
+      const ret = await fetchWithTimeout(RECOMMENDED_CHANNELS_URL, 5000)
+      if (ret.ok) {
+        const urls = (await ret.json()) as {
+          testnet: string[]
+          mainnet: string[]
+        }
+        setRecommendedChannelUrls(
+          UTIL.isMainnet() ? urls.mainnet : urls.testnet
+        )
+      } else {
+        setRecommendedChannelUrls([])
+      }
+    } catch (e) {
+      setRecommendedChannelUrls([])
+      console.error(e)
+    }
+  }
+
+  useEffect(() => {
+    fetchRecommendedChatUrls()
+    setChannels([])
+  }, [])
```

Comment:
just put this in useExploreChannels hook

Reply:
changed